front/app/containers/IdeasShow/isFieldEnabled.ts

```diff
@@ -15,3 +16,28 @@ export default function isFieldEnabled(
     ] !== 'hidden'
   );
 }
+
+// TODO: Remove when we remove associated feature flags. Temporary solution for handling both cases right now.
+export const checkFieldEnabled = (
+  fieldName: CustomFieldCodes,
+  ideaCustomFieldsSchemas: IIdeaFormSchemas | IIdeaJsonFormSchemas,
+  locale: Locale,
+  ideaCustomFieldsIsEnabled: any,
```

Comment:
We could avoid using any here and in other places. Minor: we could rename booleans to start with `is` or `has`